vroom/clangformat.vroom

```diff
@@ -180,6 +180,62 @@ that clang-format has a version >= 3.4).
 
 
 
+For clang-format versions 3.8 and above, it will sort #include lines (using the
+-sort-includes option).
+
+  :Glaive codefmt clang_format_executable='clang-format-3.7'
+  @clear
+  % #include <stdlib.h><CR>
+  |#include <stdio.h><CR>
+  |int f() { int i = 1; return 1234567890; }
+
+  :goto 1
```

Comment:
Good point.

In fact, in this test, we don't care about the cursor position at all, so I'll remove them (and replace `-cursor 0` with `-cursor \d+` in the expectation).